pytest-selenium/utils/utility.py

```diff
@@ -3,14 +3,27 @@
 
 class Library(object):
```

Comment:
i don't understand why this is a class, having to instantiate it before accessing the book utilities is annoying. can we make everything static? or lose the class and just export the functions?

Reply:
Tom, I removed the get_default_page out of the class so it no longer needs the instantiation.

I still have the book dict as a class because, I was having trouble with the book slug fixture while getting rid of the class and also while getting item values from the dict